hub/core/chunk_engine/tests/common.py

```diff
@@ -88,6 +94,14 @@ def assert_chunk_sizes(
         str(incomplete_chunk_names),
     )
 
+    # assert that all chunks (except the last one) are of expected size (`chunk_size`)
+    actual_chunk_lengths = np.array(list(actual_chunk_lengths_dict.values()))
```

Comment:
nit-pick: this function is now too long and some part of it can be broken down to smaller methods.

Reply:
i will leave it like this, mostly because the logic will need to change once we add non-constant chunk sizes.